apps/web/components/ActionButtons.tsx

```diff
@@ -56,6 +61,20 @@ export function ActionButtons(props: {
     setIsTrashing(false);
   }, [threadId, refetch]);
 
+  const [isReadUnreading, setisReadUnreading] = useState(false);
+  const [isStateRead, setIsStateRead] = useState(isRead);
+  useMemo(() => {
+    setIsStateRead(isRead);
+  }, [isRead]);
```

Comment:
Not sure what the goal is here but this isn't how `useMemo` is used.

Learn more here: https://react.dev/reference/react/useMemo